frontend/src/components/SearchTable/SearchTable.jsx

```diff
@@ -29,55 +33,65 @@ export default function SearchTable() {
                         `${response.statusText}: Please try again later.`
                     );
                 }
+                setLoading(false);
             } catch (error) {
                 setErrorMessage(`${error.message}: Please try again later.`);
             }
         };
         fetchStocks();
     }, []);
 
-    return stocksNotFound ? (
-        <p>
-            There are no stocks currently in the database, search for a stock to
-            gain access to table.
-        </p>
-    ) : (
-        <table className="search-table">
-            <tbody>
-                <tr>
-                    <th>Logo</th>
-                    <th>Ticker</th>
-                    <th>Name</th>
-                    <th>Sector</th>
-                    <th>Price</th>
-                </tr>
-            </tbody>
-            {stocks?.map((stock) => {
-                return (
-                    <tbody key={stock.name}>
+    return (
+        <>
+            {stocksNotFound ? (
+                <div
+                    class="alert alert-danger d-flex justify-content-center"
```

Comment:
class or className?

Reply:
Curious: Can the class ("alert alert-danger d-flex justify-content-center") be applied to a <p> ?
Would it be different ? Using <p> over <div> has a semantic advantage.